<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SwitchOrderDialog</class>
 <widget class="QDialog" name="SwitchOrderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>查询</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>393</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEdit_from_2"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_to_2"/>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit_2">
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2020</year>
         <month>12</month>
         <day>2</day>
        </datetime>
       </property>
       <property name="displayFormat">
        <string>yyyy-M-d</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_search_2">
       <property name="text">
        <string>车票查询</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTableWidget" name="ticketsTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>1041</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="passagerComboBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="chairComboBox">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>二等座</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>一等座</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>商务座</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>20</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resouce.qrc">:/switch.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>20</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪菱心体简</family>
     <pointsize>27</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>快速改签</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resouce.qrc"/>
 </resources>
 <connections/>
</ui>
